api/index.js

```diff
@@ -296,7 +296,7 @@ function addNewApiKey(project) {
     const projects = JSON.parse(file);
     const index = projects['active'].findIndex((item) => item['domain'] === project);
     if (index === -1) {
-      // Add new project to project list
+      // Add new project to project list  
```

Comment:
Remove rouge whitespace
```suggestion
      // Add new project to project list 
```